src/components/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Header = () => (
+  <div className="header">
+    <NavLink to="/" exact>Home</NavLink>
+     &nbsp;
+    <NavLink to="/login" exact>Login</NavLink>
```

Comment:
@tomuhenry Is the `exact` necessary here since you have used it in the Routes in `src/routes/index.js`?
It appears to be a redundancy.
Let me know what you think.